released/discovery_center/mission_3252/main.tf

```diff
@@ -0,0 +1,80 @@
+###############################################################################################
+# Setup of names in accordance to naming convention
+###############################################################################################
+resource "random_uuid" "uuid" {}
+
+locals {
+  random_uuid               = random_uuid.uuid.result
+  project_subaccount_domain = lower(replace("mission-3252-${local.random_uuid}", "_", "-"))
+}
+
+###############################################################################################
+# Creation of subaccount
+###############################################################################################
+resource "btp_subaccount" "project" {
+  name      = var.subaccount_name
+  subdomain = local.project_subaccount_domain
+  region    = lower(var.region)
+}
+
+###############################################################################################
+# Assignment of users as sub account administrators
+###############################################################################################
+resource "btp_subaccount_role_collection_assignment" "subaccount-admins" {
+  for_each             = toset("${var.subaccount_admins}")
+  subaccount_id        = btp_subaccount.project.id
+  role_collection_name = "Subaccount Administrator"
+  user_name            = each.value
+}
+
+###############################################################################################
+# Assignment of users as sub account service administrators
+###############################################################################################
+resource "btp_subaccount_role_collection_assignment" "subaccount-service-admins" {
+  for_each             = toset("${var.subaccount_service_admins}")
+  subaccount_id        = btp_subaccount.project.id
+  role_collection_name = "Subaccount Service Administrator"
+  user_name            = each.value
+}
+
+######################################################################
+# Setup Kyma
+######################################################################
+data "btp_regions" "all" {}
+
+data "btp_subaccount" "this" {
```

Comment:
This data block is not necessary as the resource got created in line 14 and the computed data can be used from this resource

Reply:
done